components/SocialShareButtons/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from 'react-share';
```

Comment:
We dont need to use external library for buttons. These are just buttons that redirect users to related accounts. We can create these components with using Html + Tailwind Css. You should uninstall package.